cmd/kubeadm/app/cmd/init.go

```diff
@@ -128,6 +128,8 @@ type initData struct {
 	dryRunDir             string
 	externalCA            bool
 	client                clientset.Interface
+	waiter                apiclient.Waiter
```

Comment:
@neolit123 if I'm not wrong the waiter is used only by the wait-control-plane phase, so IMO this can be a local variable in the phase

NB. the other usage of waiter is in self-hosting, that will go away soon